lib/chutney/TorNet.py

```diff
@@ -52,7 +53,6 @@ def getenv_type(env_var, default, type_, type_name=None):
     """
        Return the value of the environment variable 'envar' as type_,
        or 'default' if no such variable exists.
-
```

Comment:
Why did you delete all these newlines?
What editor are you using?

We try to make our diffs as small as possible, so we don't have conflicts.
So please don't change whitespace in unrelated code.